scripts/assign-colors

```diff
@@ -21,7 +21,15 @@ if __name__ == '__main__':
     with open(args.ordering) as f:
         for line in f.readlines():
             array = line.strip().split("\t")
-            if len(array) == 2:
+            # Ignore empty lines or commented lines
+            if not array or array[0].startswith('#'):
```

Comment:
I believe the `array` will still be truthy here even if the line is empty because it will have one element that is an empty string.

```suggestion
            if not array[0] or array[0].startswith('#'):
```